pkg/cli/cli.go

```diff
@@ -25,6 +25,13 @@ func Run(configPath string, inputPath string, outputPath string) error {
 			obfuscators = append(obfuscators, obfuscator.NewKeywordsObfuscator(o.Replacement))
 		case schema.ObfuscateTypeMAC:
 			obfuscators = append(obfuscators, obfuscator.NewMacAddressObfuscator())
+		case schema.ObfuscateTypeDomain:
+			// The schema does not make the default value for the TLDs a string array instead an interface array.
```

Comment:
yeah you can skip that when you supply the item types, left a comment below

Reply:
Doesn't work, already tried.